### PR TITLE
Updates to tutorial README.md files

### PR DESCRIPTION
**Pull request by philipcardiff**
I have added this pull request to make tracking the differences with the development branch easy. When ready, we can also use this to make the merge.

### CODE REVIEW

File: tutorials/solids/linearElasticity/pressurisedCylinder/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+## Results
+
+The analytical solution for the stress field is [1]:  
+$$
+\sigma_r = p \dfrac{r_i^2}{r_o^2-r_i^2}\left(1-\dfrac{r_o^2}{r^2}\right),
+$$
+
+$$
+\sigma_r = p \dfrac{r_i^2}{r_o^2-r_i^2}\left(1+\dfrac{r_o^2}{r^2}\right),
```
Comment (philipcardiff) [inline tutorials/solids/linearElasticity/pressurisedCylinder/README.md:39]: @iBatistic : can you check these formulae, I guess this should be sigma_theta

**Comment by philipcardiff** [comment]
Hi @iBatistic, this all looks great. I have reviewed all READMEs and made minor edits. Once the checks pass, they are ready for merging. It looks like some cases are failing on other versions, so we will need to fix those first (limiting cases to one version is the easiest solution for now).

**Comment by iBatistic** [comment]
@philipcardiff are you referring to failed checks on commits `6cd8c51` and `7ee0ac2`? 

They failed because of some GitHub internal server error; other commits are fine.

**Comment by philipcardiff** [comment]
Merged. I will add these tutorials to the website repository.

Feel free to continue to add to the development-README branch and we can open a new pull request for further changes.

**Comment by philipcardiff** [comment]
They are now on the website. Feel free to check them.

**Comment by iBatistic** [comment]
> Feel free to continue to add to the development-README branch and we can open a new pull request for further changes.

Sure, I will continue adding new readme files in this branch.


> They are now on the website. Feel free to check them.


Great, ok.

**Pull request merged**